src/Symfony/Component/Console/Application.php

```diff
@@ -117,10 +117,17 @@ public function run(InputInterface $input = null, OutputInterface $output = null
         $this->configureIO($input, $output);
 
         try {
+            $e = null;
             $exitCode = $this->doRun($input, $output);
-        } catch (\Exception $e) {
+        } catch (\Exception $x) {
+            $e = $x;
+        } catch (\Throwable $x) {
```

Comment:
Casting from Error (it can't be anything else) to Exception here is a behaviour change - applications that use console will find their fatal error handlers are never triggered.